plugins/modules/docker_container_copy_into.py

```diff
@@ -0,0 +1,327 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2022, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: docker_container_copy_into
+
+short_description: Copy a file into a Docker container
+
+description:
+  - Copy a file into a Docker container.
+  - Similar to C(docker cp).
+  - The file is processed in-memory. Do not use for large files!
+  - To copy files in a non-running container, you must provide the I(owner_id) and I(group_id) options.
+
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: none
+
+options:
+  container:
+    description:
+      - The name of the container to copy files to/from.
```

Comment:
```suggestion
      - The name of the container to copy files to.
```

Current module does not seem to support copying files out of the container.

Reply:
Thanks for spotting this! I originally wanted the module to support both directions, but then decided to better split that up into two modules (and start with the more interesting direction).